core/tests/upgrade-test/tests/upgrade.test.ts

```diff
@@ -214,16 +218,20 @@ describe('Upgrade test', function () {
         await utils.sleep(2);
     });
 
+    // bh ERROR
     step('Check bootloader is updated on L2', async () => {
         const receipt = await waitForNewL1Batch(alice);
-        const batchDetails = await alice.provider.getL1BatchDetails(receipt.l1BatchNumber);
+        const batchDetails = await alice.provider.getL1BatchDetails(receipt.l1BatchNumber!);
+        // bh ERROR - Property 'baseSystemContractsHashes' does not exist on type 'BatchDetails'.ts(2339)
```

Comment:
Please mark with a TODO or remove if no longer relevant, same for other ones in this file

Reply:
Yes you are right, this has been resolved with v6.9.0, I am removing outdated comments!